cdap-ui/app/cdap/components/AbstractWidget/AbstractWidgetFactory.tsx

```diff
@@ -56,6 +58,8 @@ export const WIDGET_FACTORY = {
   'connection-browser': PluginConnectionBrowser,
   csv: CSVWidget,
   'dataset-selector': DatasetSelector,
+  'date-range-widget': DateRangeWidget,
```

Comment:
Just `daterange` and `datetime` would be fine. We don't have widget in any other widget's name.